packages/webapp/pages/onboarding.tsx

```diff
@@ -414,16 +415,11 @@ export function OnboardPage(): ReactElement {
               reverse
               className=" mt-5 block tablet:hidden"
             />
-
-            <img
-              className="absolute bottom-0 left-0 -z-1 w-full max-w-[58.75rem]"
-              src={cloudinary.onboarding.glow}
-              alt="Gradient background"
-            />
           </div>
           <div className="hidden flex-1 tablet:block" />
         </footer>
       )}
+      <FooterLinks className="mx-auto" />
```

Comment:
![image](https://github.com/dailydotdev/apps/assets/1681525/ad7123d0-a6bf-4c54-a085-34b4747c3ef1)
Missing some bottom padding on onboarding page?

Reply:
I know on laptop screens the image is a bit buggy and causes vertical scroll. It's something we should fix now that we've removed video support, we can fix the scaling issues.

Reply:
yeah, thats why I missed the padding, added now